extensions/vault/runtime/pom.xml

```diff
@@ -19,8 +19,8 @@
             <artifactId>quarkus-core</artifactId>
         </dependency>
         <dependency>
-            <groupId>com.squareup.okhttp3</groupId>
-            <artifactId>okhttp</artifactId>
+            <groupId>io.smallrye.reactive</groupId>
+            <artifactId>smallrye-mutiny-vertx-web-client</artifactId>
```

Comment:
Just to be sure, this one is working in native without any needs for an extension?

Reply:
Yes, works flawlessly in native.